packages/fluentui/docs/src/examples/components/Dropdown/State/DropdownExampleDisabled.shorthand.tsx

```diff
@@ -0,0 +1,15 @@
+import { Dropdown, Flex } from '@fluentui/react';
+import * as React from 'react';
+
+const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange', 'Alfred Pennyworth'];
+
+const DropdownExampleDisabled = () => {
+  return (
+    <Flex column gap="gap.smaller">
+      <Dropdown disabled={true} items={inputItems} placeholder="Select your hero, if you can ..." />
+      <Dropdown disabled={true} search={true} items={inputItems} placeholder="Start typing a name, if you can ..." />
```

Comment:
```suggestion
      <Dropdown disabled search={true} items={inputItems} placeholder="Start typing a name, if you can ..." />
```

iirc @layershifter likes it this way